extra/utils/Validation.js

```diff
@@ -0,0 +1,54 @@
+let users = [
+    {
+        traineeEmail : 'shashank.baranawal@successive.tech',
+        reviewerEmail : 'naman.parashar@successive.tech',
+    },
+    {
+        traineeEmail : 'shashank.baranwal@gmail.com',
+        reviewerEmail : 'naman.parashar@gmail.com'
+    }
+]
+
+/*function validateEmail(email){
+    const regix = /\w+.\w+@successive.tech$/i;
+    return regix.test(email)
+}*/
+
+function validateEmail(email){
+    const regex = /\w+.\w+@successive.tech$/i;
+    if(regex.test(email))
+    {
+        return true;
+    }
+    else{
+        return false;
+    }
+}
+//console.log(validateEmail(users[0]["traineeEmail"]))
+
+function validateUsers(users){
+    let countValid=0;
+    let countInvalid=0;
+    users.forEach(item => {
+        const {traineeEmail,reviewerEmail}= item;
+        if(validateEmail(item.traineeEmail)=== true && validateEmail(item.reviewerEmail)===true){
```

Comment:
Use values for de-structured object. Used destructuring but still used object dot method to access

Reply:
ya i resolved the problem.